src/seedu/addressbook/data/person/Email.java

```diff
@@ -11,7 +11,7 @@
     public static final String EXAMPLE = "valid@e.mail";
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
             "Person emails should be 2 alphanumeric/period strings separated by '@'";
```

Comment:
I think you'll need to update the constraint message as well that you need the tld.